scripts/build_gap.sh

```diff
@@ -1,4 +1,14 @@
 #!/usr/bin/env bash
+#
+# DO NOT EDIT THIS FILE!
+#
+# If you need to build additional packages, override the GAP_PKGS_TO_BUILD
+# environment variable instead, or do it from your .travis.yml build script.
+#
```

Comment:
First, about `or do it` : isn't your vision that only `.travis.yml` should be modified, and not `build_gap.sh`? Moreover, suggesting to override it here contradicts to "DO NOT EDIT" warning. 

Second, what about adding a commented out line to .travis.yml showing how to override it, and some explanatory note? Like e.g. in https://github.com/gap-packages/io/blob/master/.travis.yml

Reply:
The `.travis.yml` build script is the "script" section of `.travis.yml`, I see no contradiction here.

I do not wish to add such a comment out line to `.travis.yml`. I will however suggest that looking at the `cvec` package gives an example of how to do it. (The io package is a not a good example for this). Anyway, I do not want to add *too* much hand holding here -- at some point, people should either understand the scripts they modify, or *NOT TOUCH THEM* and instead ask for explanations.

Reply:
I see contradiction because I understand the suggestion as overriding it in `build_gap.sh`. Clearly specifying it in `.travis.yml`  is not contradictory.